src/coreclr/jit/emit.cpp

```diff
@@ -3922,6 +3920,38 @@ void emitter::emitDispJumpList()
 
 #endif // DEBUG
 
+//------------------------------------------------------------------------
+// emitAdvanceInstrDesc:
+//   Advance to the next instrDesc in the buffer of instrDescs, taking optional
+//   debug info into account.
+//
+// Parameters:
+//   id - the pointer to the current instrDesc
+//   idSize - the size of the current instrDesc
+//
+void emitter::emitAdvanceInstrDesc(instrDesc** id, size_t idSize)
+{
+    assert(idSize == emitSizeOfInsDsc(id));
```

Comment:
Don't you need a deref?
```suggestion
    assert(idSize == emitSizeOfInsDsc(*id));
```

Reply:
Yep...